src/otx/core/model/base.py

```diff
@@ -1041,14 +1041,25 @@ def export(
             msg = "OpenVINO model can be exported only as exportable code with demo package."
             raise RuntimeError(msg)
 
-        return self._exporter.export(
-            self.model,
-            output_dir,
-            base_name,
-            export_format,
-            precision,
-            to_exportable_code,
-        )
+        # Temporarily unwrap Tiler model if applicable
+        original_model = self.model
+        if isinstance(original_model, Tiler):
+            self.model = original_model.model
+
+        try:
+            exported_path = self._exporter.export(
+                self.model,
```

Comment:
Do we have to replace `self.model`? does export not work when we pass `original_model.model`?

Reply:
We need to merge this PR for 2.4 release and I will resolve this conversation. @eugene123tw could you address this question?

I think it is needed since we should cover not only Tiling case, but also "ordinary" models.